packages/config/lib/index.ts

```diff
@@ -1,206 +1,101 @@
 import * as path from "path";
 import * as _ from "lodash";
-import { randomBytes } from "crypto";
 
-// DI
 import { Inject, Service, Container } from "@fullstack-one/di";
 import { BootLoader } from "@fullstack-one/boot-loader";
 
+import { DefaultConfigNotFoundError } from "./errors";
+import ConfigMergeHelper from "./helpers/ConfigMergeHelper";
+import EnvironmentBuilder from "./helpers/EnvironmentBuilder";
 import { IEnvironment } from "./IEnvironment";
-export { IEnvironment };
 
-interface IConfigModule {
-  name: string;
-  path: string;
-}
+export { IEnvironment };
 
 @Service()
 export class Config {
-  // DI
   @Inject((type) => BootLoader)
```

Comment:
I would suggest to move this injection into the constructor for consistency reasons.